src/app/components/hris/employees/employee-approvals/view-starter-kit-approval/view-starter-kit-approval.component.html

```diff
@@ -19,8 +19,8 @@
         <div id="approval-header" class="row p-2">
             <div class="col-2 mt-3 mb-3">
                 <div id="profile-image">
-                    <img [src]="employee?.photo == '' || employee?.photo == 'TBA' ? 
-                        '../../../../../assets/img/default-profile-image.png' : employee?.photo"
+                    <img [src]="authAccessService.getAuthTokenProfilePicture() || '../../../../../../assets/img/default-profile-image.png'"
+          alt="Profile img" class="rounded-circle profile-img">
```

Comment:
Fix: the logic here should be the same as in the others -

1) first use employee profile photo, check if available
2) if employee profile photo not available, use google photo
3) if google photo not available use static image